ingest/ingest.go

```diff
@@ -0,0 +1,56 @@
+package ingest
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+const (
+	resourcePath = "/log/ingest"
+	method       = "POST"
+)
+
+type Log struct {
+	Message    string            `json:"msg"`
+	Timestamp  time.Time         `json:"timestamp"`
+	ResourceId map[string]string `json:"_lm.resourceId"`
+}
+
+type Ingest struct {
+	HostUrl   string
```

Comment:
I notice that we use:
- LM Account name (AWS) 
- Company name (public API)
- Company name (Fluentd)
- Company name (Azure)
- HostUrl (here)

How about we use CompanyName here too.